ihatemoney/web.py

```diff
@@ -269,6 +269,11 @@ def home():
         session=session,
     )
 
+@main.route("/download-mobile-app", methods=["GET"])
```

Comment:
Should this URL be easier to remember? What about something like just `/mobile`?

Reply:
Yes done.

Reply:
I don't see the changes in what you pushed. Can you check it? Thanks!